app/schemas/bean.pwr.imskamieskiego.data.LocalDB/1.json

```diff
@@ -249,11 +249,69 @@
             ]
           }
         ]
+      },
+      {
+        "tableName": "locations_points",
```

Comment:
Locations_points zmieniłbym na Location_Point. Ze względu na fakt, że każda lokalizacja może mieć jeden centralny węzeł to żadna lokacja nie powinna dwa razy pojawić się w tej tabeli z różnym punktem. W dokumentacji też nie jest to oczywiste, że w tej tabeli zawiera się centralny punkt lokacji.

Reply:
Mówisz o której dokumentacji? O java_docsach czy o wiki projektu? W wiki projektu zostało to raczej opisane, poza tym mamy tam diagram encji który pokazuje, że między tabelami locations, a locations_points mamy relację 1:1. Co do samej nazwy, to raczej przyjęło się (a przynajmniej ja przyjąłem ;) ), by table nazywać przy pomocy liczby mnogiej. W diagramie z koleii to są nazwy encji, więc powinny być w liczbie pojedyńczej (ale niestety nie wszystkie są. Muszę wprowadzić korektę do dokumentacji).

Reply:
Źle ująłem, z diagramu nie widać od razu, że to centralny punkt lokacji, ja od razu myślałem, że to tabela z punktami, które należą do lokacji.Co do do liczby mnogiej to jak najbardziej. Po prostu sama nazwa jest myląca. Może points na center_points albo po prostu center. Nie upieram się przy tym, tylko wiem, że mnie nazwa trochę zmyliła, nie wiem jak innych :D